wagtail/snippets/action_menu.py

```diff
@@ -77,6 +78,110 @@ def is_shown(self, context):
         )
 
 
+class SubmitForModerationMenuItem(ActionMenuItem):
+    name = "action-submit"
+    label = _("Submit for moderation")
+    icon_name = "resubmit"
+
+    def is_shown(self, context):
+        if not getattr(settings, "WAGTAIL_MODERATION_ENABLED", True):
+            return False
+
+        if context.get("locked_for_user"):
+            return False
+
+        if context["view"] == "create":
+            return context["model"].get_default_workflow() is not None
+
+        return (
+            not context.get("locked_for_user")
```

Comment:
Could we be showing this on the revisions_revert view here, as the corresponding page logic suggests?

Reply:
Correct, we shouldn't show it, I missed this. Thanks!